Mathlib/Algebra/Group/WithOne/Defs.lean

```diff
@@ -127,7 +127,7 @@ def recOneCoe {C : WithOne α → Sort*} (h₁ : C 1) (h₂ : ∀ a : α, C a) :
 -- would automatically get this; right now in Lean 4...I don't
 -- know if it does or not, and I don't know how to check, so
 -- I'll add it manually just to be sure.
-attribute [elab_as_elim] WithZero.recZeroCoe
+attribute [elab_as_elim, induction_eliminator] WithZero.recZeroCoe
```

Comment:
Do you now know for sure that `induction_eliminator` is not being added by `to_additive`? If so then you could add this information to the comment above.

Reply:
No, this whole comment can be removed, I just didn't want to scope creep here